index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    	<meta charset="UTF-8">
+	<title>Hello</title>
+	<link rel="stylesheet" href="./style.css">
+    </head>
+    <body>
+        <header></header>
```

Comment:
- [ ] Kindly make sure you add a `class` to the `h1` 🙏

Reply:
Thanks! It's done